packages/completer/src/manager.ts

```diff
@@ -197,6 +223,46 @@ export class CompletionProviderManager implements ICompletionProviderManager {
     }
   }
 
+  /**
```

Comment:
I don't see any issues with the new API. Only a question to keep a trace, do we want arguments list like the current proposal or an arguments object?

I'm mainly asking especially for the `cycleInline` in light of the mistake I did when refactoring the search.

Reply:
Good point, there is a place in the `IModel` where it seems more relevant (but then it is further away from the extension point). I do not see it for `cycleInline` specifically, can you elaborate? The future possibilities for extension that I see could be `direction: 'first' | 'last'`.

Reply:
I was not seeing a specific case for `cycleInline`. It was more to be sure to think a second times about that; as you did :wink: